src/Pool.sol

```diff
@@ -336,21 +323,21 @@ abstract contract Pool {
         collected += _flushCyclesInternal(receiverAddr, cycles);
 
         // Dripped when collected
-        if (collected > 0 && currReceivers.length > 0 && sender.dripsFraction > 0) {
-            uint256 drippable = (uint256(collected) * sender.dripsFraction) / MAX_DRIPS_FRACTION;
-            uint128 totalAmtPerSec = _totalAmtPerSec(currReceivers);
-            uint128 drippedAmtPerSec = 0;
+        if (collected > 0 && currReceivers.length > 0) {
+            uint32 drippedWeight = 0;
             for (uint256 i = 0; i < currReceivers.length; i++) {
-                drippedAmtPerSec += currReceivers[i].amtPerSec;
-                uint128 dripAmt = uint128((drippable * drippedAmtPerSec) / totalAmtPerSec) -
-                    dripped;
+                drippedWeight += currReceivers[i].weight;
+                uint128 dripAmt = uint128(
+                    (uint160(collected) * drippedWeight) / TOTAL_DRIPS_WEIGHTS - dripped
```

Comment:
I like it. It will also simplify the usage. I want to give 1% of my funds to project X and 5% to project Y.

Instead of first defining a dripFraction and after that the percentages. 👍